cv-data-service-library/src/main/java/com/trihydro/library/helpers/TimGenerationHelper.java

```diff
@@ -209,16 +206,14 @@ public List<ResubmitTimException> updateAndResubmitToOde(
                     exceptions.addAll(extraEx);
                 }
             } catch (Exception ex) {
-                utility.logWithDate("Failed attempting to update TIM (active_tim_id " +
-                    validationResult.getActiveTim().getActiveTimId() +
-                    ") with ActiveTimValidationResult: " + gson.toJson(validationResult));
-                ex.printStackTrace();
+                log.error(
+                    "Failed attempting to update TIM (active_tim_id {}) with ActiveTimValidationResult: {}",
```

Comment:
suggestion(non-blocking): simplify log language 
```suggestion
                    "Failed to update TIM (active_tim_id {}) with ActiveTimValidationResult: {}",
```

Reply:
Committed your suggestion!